lago/workdir.py

```diff
@@ -503,3 +502,18 @@ def set_current(prefix_name, parent_workdir, **kwargs):
         workdir(str): path to the workdir to change the current of
     """
     parent_workdir.set_current(new_current=prefix_name)
+
+
+class LagoPrefixAlreadyExistsError(utils.LagoException):
+    def __init__(self, name, path):
+        super().__init__(
+            dedent(
+                """
+                Failed Prefix name {} already exists in workdir {}
+                1) Destroy old lago VM in order to create new VM
```

Comment:
I don't think that we should specify command names and flags here, because if the name of the command (or its flags) will change, we will need to remember change this message.

A good example is the `WORKDIR-PATH` positional parameter that we think to remove.